back/engines/commercial/analytics/app/services/analytics/query_schema.json.erb

```diff
@@ -24,7 +24,7 @@
     },
     "filters" : {
       "type" : "object",
-      "minProperties" : 1,
+      "minProperties" : 0,
```

Comment:
Disabled this, because sometimes you have a query like

```
filters: {
    **project_filter('dimension_project_id', project_id),
 },
```

but there is no `project_id`, which results in an empty filters object. Functionally that seems to behave fine